src/main/kotlin/org/jitsi/nlj/srtp/SrtpConfig.kt

```diff
@@ -17,11 +17,17 @@ package org.jitsi.nlj.srtp
 
 import org.jitsi.config.JitsiConfig
 import org.jitsi.metaconfig.config
+import org.jitsi.nlj.srtp.SrtpUtil.Companion.getSrtpProtectionProfileFromName
 
 class SrtpConfig {
     companion object {
         val maxConsecutivePacketsDiscardedEarly: Int by config {
             "jmt.srtp.max-consecutive-packets-discarded-early".from(JitsiConfig.newConfig)
         }
+
+        val protectionProfiles: List<Int> by config {
```

Comment:
Do we access this anywhere else?  If not I would just make it a member of where it's used (I'd say the same for `maxConsecutivePacketsDiscardedEarly`, if it applies)

Reply:
We access it both from `TlsClientImpl` and `TlsServerImpl`.  These don't share a common base class outside BouncyCastle.

Putting it in `SrtpConfig`, i.e. under `org.jitsi.nlj.srtp` rather than under `org.jitsi.nlj.dtls` is a little weird, but I think that makes more semantic sense for how it's used, regardless of where the code that uses it is.  Similarly, putting its config under `jmt.srtp` rather than `jmt.dtls`.